app/controllers/games_controller.rb

```diff
@@ -1,5 +1,18 @@
 class GamesController < ApplicationController 
 	def show 
 		render template: "games/#{params[:game]}"
-	end
+	end 
+
+	def new 
+		@game = Game.new 
+	end 
+
+	def create 
+		@game = Game.create 
+	end 
+
+	def update 
+		@game = Game.find(params[:id]) 
+		current_player.update_attribute(:game_id, @game.id) 
```

Comment:
Illiana, I have the new/create methods started in PR #17 as part of the Game CRUD ticket. Willem, what you suggested makes sense, which makes me think I need to do the same in the create method, since whomever creates a new game would be the first user/player...thoughts?